src/value/use-transform.ts

```diff
@@ -160,12 +160,19 @@ export function useTransform<I, O>(
  * @public
  */
 export function useTransform<I, O>(
-    input: MotionValue<string | number>[],
+    input:
```

Comment:
you might consider assigning this typings to a type variable and reuse it in the next function below